profile-widget/diveeventitem.cpp

```diff
@@ -152,17 +152,21 @@ void DiveEventItem::setupToolTipString()
 
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
-			name += QString(" (cyl %1)").arg(internalEvent->gas.index+1);
+			name += tr(" (cyl.") + QString(" %1)").arg(internalEvent->gas.index+1);
```

Comment:
why don't you write this as 
tr(" cyl. %1").arg(internalEvent->gas.index + 1); ?

(Note also the spaces around + which were missing already in old version)